Ginger/Ginger/App.xaml.cs

```diff
@@ -332,8 +341,30 @@ private bool ShouldStartGrid(string[] args)
         /// <returns>ParserResult containing parsed arguments or null.</returns>
         private ParserResult<object> ParseCommandLineArguments(string[] args)
         {
+            string[] arguments;
+            if (args.Length == 1 && System.Web.HttpUtility.UrlDecode(args[0]).Equals("ginger:///", StringComparison.OrdinalIgnoreCase))
+            {
+                return null;
+            }
+            if (args.Length==1)
+            {
+                string input = args[0];
+                input = System.Web.HttpUtility.UrlDecode(input);
+                if (input.StartsWith("ginger://"))
+                {
+                    input = input.Substring("ginger://".Length);
+                }
+                List<string> resultList = SplitWithPaths(input).Select(s => s.Trim('\"')).ToList();
+                arguments = resultList.ToArray();
+            }
+            else
+            {
+                arguments = args;
+            }
+            EditCommandlineArguments(arguments);
```

Comment:
do we need EditCommandlineArguments after UrlDecode on 352?

Reply:
no,removed